packages/rest/.npmrc

```diff
@@ -0,0 +1 @@
+package-lock=false
```

Comment:
Package-local npmrc duplicating server-wide config does not make sense to me, can we remove this file?